operations/network/dns/Manage_the_DNS_Unbound_Resolver.md

```diff
@@ -150,3 +150,76 @@ ncn-w001# kubectl -n services patch configmaps cray-dns-unbound \
 --type merge -p '{"binaryData":{"records.json.gz":"H4sICLQ/Z2AAA3JlY29yZHMuanNvbgCLjuUCAETSaHADAAAA"}}'
 ```
 
+### Change the site DNS server
+
+Use the following procedure to change the site DNS server that Unbound forwards queries to.
```

Comment:
It could by helpful to add more context here as to why an administrator would need to change this post-install.

Reply:
Added additional context